web/src/fileview/fileview.js

```diff
@@ -28,6 +32,80 @@ function doSearch(event, query, newTab) {
   }
 }
 
+function isInCodeScope(node) {
```

Comment:
This looks a bit like the jQuery call `closest('#source-code')` — is jQuery available at this point, and if so, would that substitute for this one or is there a difference that would prevent it from working?

Reply:
Good point!